<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="3553" started-at="2019-06-04T21:19:17Z" finished-at="2019-06-04T21:19:20Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="3553" started-at="2019-06-04T21:19:17Z" finished-at="2019-06-04T21:19:20Z">
      <class name="Selenium.demoaut">
        <test-method status="PASS" signature="runningTest()[pri:0, instance:Selenium.demoaut@1c7e1bb]" name="runningTest" is-config="true" duration-ms="3780" started-at="2019-06-04T21:19:13Z" finished-at="2019-06-04T21:19:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runningTest -->
        <test-method status="PASS" signature="launcher()[pri:0, instance:Selenium.demoaut@1c7e1bb]" name="launcher" is-config="true" duration-ms="1890" started-at="2019-06-04T21:19:17Z" finished-at="2019-06-04T21:19:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launcher -->
        <test-method status="PASS" signature="url()[pri:0, instance:Selenium.demoaut@1c7e1bb]" name="url" is-config="true" duration-ms="0" started-at="2019-06-04T21:19:19Z" finished-at="2019-06-04T21:19:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- url -->
        <test-method status="PASS" signature="login()[pri:0, instance:Selenium.demoaut@1c7e1bb]" name="login" duration-ms="1483" started-at="2019-06-04T21:19:19Z" finished-at="2019-06-04T21:19:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="FAIL" signature="titlecheck()[pri:0, instance:Selenium.demoaut@1c7e1bb]" name="titlecheck" duration-ms="27" started-at="2019-06-04T21:19:20Z" finished-at="2019-06-04T21:19:20Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<http://newtours.demoaut.com/mercurysignon.php?osCsid=4a72614ef4a62c7ba5ae51999718c0c8> but was:<http://newtours.demoaut.com/mercurysignon.php?osCsid=2d7c84bb8da9d5ecd5c45954931c1edf>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<http://newtours.demoaut.com/mercurysignon.php?osCsid=4a72614ef4a62c7ba5ae51999718c0c8> but was:<http://newtours.demoaut.com/mercurysignon.php?osCsid=2d7c84bb8da9d5ecd5c45954931c1edf>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
at Selenium.demoaut.titlecheck(demoaut.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titlecheck -->
        <test-method status="PASS" signature="closetab()[pri:0, instance:Selenium.demoaut@1c7e1bb]" name="closetab" is-config="true" duration-ms="0" started-at="2019-06-04T21:19:20Z" finished-at="2019-06-04T21:19:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closetab -->
        <test-method status="PASS" signature="close()[pri:0, instance:Selenium.demoaut@1c7e1bb]" name="close" is-config="true" duration-ms="114" started-at="2019-06-04T21:19:20Z" finished-at="2019-06-04T21:19:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="PASS" signature="closeTest()[pri:0, instance:Selenium.demoaut@1c7e1bb]" name="closeTest" is-config="true" duration-ms="1" started-at="2019-06-04T21:19:21Z" finished-at="2019-06-04T21:19:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeTest -->
      </class> <!-- Selenium.demoaut -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
